src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {

```

Comment:
According to the encapsulation principle and best practices, the class should have an access modifier. Consider using `public` for this abstract class to ensure it can be accessed from other packages if needed.